src/magql/resolver_factory.py

```diff
@@ -163,6 +165,11 @@ def __call__(self, parent, info, *args, **kwargs):
         return parent
 
 
+class CountResolver:
+    def __call__(self, parent, info, *args, **kwargs):
+        return info.context.info.get("count")
```

Comment:
This isn't portable to non-SQLAlchemy setups. We should look into creating a dedicated info/context rather than treating the SQLAlchemy session as that.